.github/workflows/CI.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm install --save-dev --save-exact prettier
+      - run: npx prettier --check .
```

Comment:
Also, we don't need line 23 since we're already installing required packages on line 22
Also, on line 24, we just declare what needs to be run from the scripts, let's not create anything here. It should just be `npm run check` here. It's the package json's job to have defined what's `check` script
I hope that makes sense